README.md

```diff
@@ -22,12 +22,12 @@ To read about the backstory on why we created RIBs, see [this blog post](https:/
 
 #### What is the difference between RIBs and MV*/VIPER?
 
-MVC, MVP, MVVM and VIPER are architecture patterns. RIBs is a framework. What differentiates RIBs from these patterns is these:
+MVC, MVP, MVVM, MVI and VIPER are architecture patterns. RIBs is a framework. What differentiates RIBs from frameworks based on MV*/VIPER is:
```

Comment:
Small nit, can we move `MVI` before `MVVM`

Reply:
Sure.